src/core/logic.py

```diff
@@ -1221,3 +1221,58 @@ def create_organization_name(request):
             % {"organization": organization_name},
         )
         return organization_name
+
+
+def get_contact_form(request, recipient_uuid):
+    subject = request.GET.get("subject", "")
+    contact_people = request.site_type.contact_people
+    account = None
+    if recipient_uuid:
+        try:
+            account = models.Account.objects.get(
+                contactperson__in=contact_people,
+                uuid=recipient_uuid,
+            )
```

Comment:
If I know the UUID of another account, I think this potentially allows an attacker to get a contact from from a different site. Instead, and since `request` is an argument to this function, we can flip the query to retrieve a `Contact` person and match against the `request.site_type`